app/sbna-poc/v1_0__initial-form/fields/thinking-behaviours-attitudes.ts

```diff
@@ -0,0 +1,521 @@
+  },
+  {
+    text: 'Does [subject] support or excuse criminal behaviour?',
+    code: 'thinking_behaviours_attitudes_criminal_behaviour',
+    type: FieldType.Radio,
+    validate: [{ type: ValidationType.Required, message: 'Select if they support or excuse criminal behaviour' }],
+    options: [
+      {
+        text: 'Yes, supports or excuses criminal behaviour or their pattern of behaviour and other evidence indicates this is an issue',
+        value: 'YES_SUPPORTS_CRIMINAL_BEHAVIOUR',
+        kind: 'option',
+      },
+      {
+        text: 'Sometimes supports or excuses criminal behaviour',
+        value: 'SOMETIMES_SUPPORTS_CRIMINAL_BEHAVIOUR',
+        kind: 'option',
+      },
+      {
+        text: 'No, does not support or excuse criminal behaviour',
+        value: 'NOT_SUPPORT_CRIMINAL_BEHAVIOUR',
+        kind: 'option',
+      },
+    ],
+    labelClasses: getMediumLabelClassFor(FieldType.Radio),
+  },
+]
+
+const makeChangesOptionsWithDetails: Array<FormWizard.Field.Option> = [
+  { text: 'I have already made changes and want to maintain them', value: 'MADE_CHANGES', kind: 'option' },
+  { text: 'I am actively making changes', value: 'MAKING_CHANGES', kind: 'option' },
+  { text: 'I want to make changes and know how to', value: 'WANT_TO_MAKE_CHANGES', kind: 'option' },
+  { text: 'I want to make changes but need help', value: 'NEEDS_HELP_TO_MAKE_CHANGES', kind: 'option' },
+  { text: 'I am thinking about making changes', value: 'THINKING_ABOUT_MAKING_CHANGES', kind: 'option' },
+  { text: 'I do not want to make changes', value: 'DOES_NOT_WANT_TO_MAKE_CHANGES', kind: 'option' },
+  { text: 'I do not want to answer', value: 'DOES_NOT_WANT_TO_ANSWER', kind: 'option' },
+]
+
+const makeChangesOptions: FormWizard.Field.Options = [
+  ...makeChangesOptionsWithDetails,
+  orDivider,
+  { text: '[subject] is not present', value: 'NOT_PRESENT', kind: 'option' },
+  { text: 'Not applicable', value: 'NOT_APPLICABLE', kind: 'option' },
+]
+
+export const makeChangesFields: Array<FormWizard.Field> = [
+  {
+    text: 'Does [subject] want to make changes to their thinking behaviours and attitudes?',
+    hint: { text: 'This question must be directly answered by [subject]', kind: 'text' },
+    code: 'thinking_behaviours_attitudes_changes',
+    type: FieldType.Radio,
+    validate: [
+      {
+        type: ValidationType.Required,
+        message: 'Select if they want to make changes to their thinking, behaviours and attitudes',
+      },
+    ],
+    options: makeChangesOptions,
+    labelClasses: getMediumLabelClassFor(FieldType.Radio),
+  },
+  ...makeChangesOptionsWithDetails.map(detailsFieldWith('thinking_behaviours_attitudes_changes')),
+]
+
+export const practitionerAnalysisFields: Array<FormWizard.Field> =
+  createPractitionerAnalysisFieldsWith('health_wellbeing')
+
+export const questionSectionComplete: FormWizard.Field = {
+  text: 'Is the health and wellbeing section complete?',
+  code: 'health_wellbeing_section_complete',
```

Comment:
I think this should be `thinking_behaviours_attitudes_section_complete`

Reply:
ah yes, thanks!